app/templates/signup.hbs

```diff
@@ -1,74 +1,48 @@
 {{page-title "Sign Up"}}
 
-{{#if this.dev}}
-  {{#if this.state}}
-    {{#if (eq this.state "get-started")}}
-      <SignUp::GetStarted
-        @changeRouteParams={{this.changeRouteParams}}
-        @state={{this.state}}
-      />
-    {{else if (eq this.state "thank-you")}}
-      <SignUp::GetStarted
-        @changeRouteParams={{this.changeRouteParams}}
-        @state={{this.state}}
-      />
+<form class="form">
+  <h2>{{this.title}}</h2>
+  {{#each this.fields as |field|}}
+  {{#let this.formData as | myFormData |}}
+    <FormInput
+      @id={{field.id}}
+      @for={{field.for}}
+      @label={{field.label}}
+      @type={{field.type}}
+      @value={{get myFormData field.id}}
+      @placeholder={{field.placeholder}}
+      @errorMessage={{field.errorMessage}}
+      @required={{field.required}}
+      @showError={{field.showError}}
+      @helpMsg={{field.helpMsg}}
+      @onChange={{this.handleFieldChange}}
+      @validator={{field.validator}}
+    />
+    {{/let}}
+  {{/each}}
+  {{#if this.isSubmitDisabled }}
```

Comment:
Will this not just hide the button completely? (Instead of decreasing its opacity)